db/schema.rb

```diff
@@ -297,6 +297,7 @@
   add_index "course_assessment_question_programming_template_files", ["question_id", "filename"], :name=>"index_course_assessment_question_programming_template_filenames", :unique=>true, :case_sensitive=>false
 
   create_table "course_assessment_question_text_responses", force: :cascade do |t|
+    t.boolean "allow_attachment", :default=>false
```

Comment:
This is a child table, but everything is same as its parent, that's why empty.

It's something related to the act_as gem